### PR TITLE
Add studios and languages registries and generalize validate.js

### PR DESCRIPTION
**Pull request by radford-for-smpte**

### CODE REVIEW

File: src/main/scripts/validate.js [hunk 1]
```diff
@@ -19,45 +19,62 @@ CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING
...
-  throw "Registry fails validation";
-};
-
-/* is the registry sorted */
-
-for(let i = 1; i < registry.length; i++) {
-  if (registry[i-1].code >= registry[i].code) {
-    throw "Registry key " + registry[i-1].code + " is " +
-      ((registry[i-1].code === registry[i].code) ? "duplicated" : "not sorted");
+  if (name ==='studios' || name === 'facilities') {
```
Comment (radford-for-smpte) [inline src/main/scripts/validate.js:44]: @palemieux curious if you think these registry-specific checks should be moved to their own module, e.g. `<name>-validate.js`.

Comment (palemieux) [reply src/main/scripts/validate.js:44]: You read my mind.

Comment (radford-for-smpte) [reply src/main/scripts/validate.js]: Take a look at the latest.

**Comment by palemieux** [comment]
@radford-for-smpte LGTM. Do you think we should wait on completing the HTML rendering at https://github.com/ISDCF/registries-site/issues/4 before merging?

**Comment by radford-for-smpte** [comment]
> @radford-for-smpte LGTM. Do you think we should wait on completing the HTML rendering at [ISDCF/registries-site#4](https://github.com/ISDCF/registries-site/issues/4) before merging?

I'd think you'd start with `registries`, and then do the `registries-site` afterwards. What I'm personally most interested I seeing `ISDCF/registries-server` function with more than one registry.

**Comment by SteveLLamb** [comment]
> > @radford-for-smpte LGTM. Do you think we should wait on completing the HTML rendering at [ISDCF/registries-site#4](https://github.com/ISDCF/registries-site/issues/4) before merging?
> 
> I'd think you'd start with `registries`, and then do the `registries-site` afterwards. What I'm personally most interested I seeing `ISDCF/registries-server` function with more than one registry.

Agreed @radford-for-smpte, get the JSON's and that logic all built out, we can build site HTMLS out as that goes.

**Pull request merged**